test/README.md

```diff
@@ -12,11 +12,12 @@ On how to run tests in this directory, see
 
 |Directory          |Runs on CI     |Purpose        |
 |-------------------|---------------|---------------|
-|abort              |No             |Tests for when the  ``` --abort-on-uncaught-exception ``` flag is used.|
+|abort              |Yes            |Tests for when the  ``` --abort-on-uncaught-exception ``` flag is used.|
 |addons             |Yes            |Tests for [addon](https://nodejs.org/api/addons.html) functionality along with some tests that require an addon to function  properly.|
 |cctest             |Yes            |C++ test that is run as part of the build process.|
 |common             |               |Common modules shared among many tests. [Documentation](./common/README.md)|
 |debugger           |No             |Tests for [debugger](https://nodejs.org/api/debugger.html)functionality along with some tests that require an addon to function properly.|
```

Comment:
It is out of scope for this PR, but maybe it is appropriate to add a space before "functionality" here while you are at this?